frontend/src/app/person/person.component.spec.ts

```diff
@@ -128,7 +142,7 @@ describe('PersonComponent', () => {
     fixture.detectChanges();
 
     const nativeElement = fixture.nativeElement;
-    const mediationDependantElementIds = ['entryDate', 'mediationCode', 'maritalStatus', 'housing', 'fiscalStatus', 'healthCareCoverage'];
+    const mediationDependantElementIds = ['entryDate', 'mediationCode', 'firstMediationAppointmentDate', 'maritalStatus', 'healthCareCoverage'];
```

Comment:
why have the housing and fiscalStatus IDs disappeared?

Reply:
inadvertent error